src/maskAnalysis/getFeret.ts

```diff
@@ -73,7 +99,7 @@ export function getFeret(mask: Mask): Feret {
   let minWidth = Number.POSITIVE_INFINITY;
   let minWidthAngle = 0;
   let minLinePoints: Point[] = [];
```

Comment:
We only need during the search for the minFeretDiameter to keep the minX and maxX in order to calculate the minCalliperLines. You can name this variable minCalliperXs.